packages/components/src/HeaderBar/HeaderBar.scss

```diff
@@ -78,6 +78,18 @@ $background-color-on-hover: shade($brand-primary, 30);
 			font-size: $font-size-base;
 			text-transform: uppercase;
 		}
+
+		.tc-header-bar-call-to-action {
+			:global(.btn):global(.btn-info):global(.btn-inverse) {
```

Comment:
I would prefer something like that, WDYT ?

```css
:global {
    .btn.btn-info.btn-inverse {
        // ...
    }
}
```

Reply:
I think it's less verbal and more readable, without redundant :global